TMP2022/technical-writing-track/Ruth-tw-Moses/README.md

```diff
@@ -0,0 +1,43 @@
+OVERVIEW This is a guide to users who are just getting started with the kuda bank app. The app lets users transfer receive and save money conveniently with little charges.
+The steps below will guide you on how to get started with the app.
+
+Install the App Step 1: Visit the Google play store app on your mobile device and download the Kuda bank app.
+Step 2: Once you have installed the app, click on open to get started.
+
+Create An Account Step 1: Click on Open An Account.
+Step 2: input a valid email address.
+
+Step 3: Enter a password you will remember.
+
+Step 4: Confirm your password, which is the one you input above.
+
+Step 5: Input a referral code if any.
+
+Step 5: Read the terms and conditions and agree to them by clicking on "next".
+
+N/B:you will be sent a verification code to the email address you input above, so be vigilant.
+
+Step 6: Input the Six digits code sent to your Gmail address and click " Next".
+
+N/B By clicking on "Next", it takes you to the next page which requires you to fill in information about yourself, please fill them in as they appear on your official documents.
+
+Step 7: Input your Name, middle name, last name, Gender, And Date of Birth in their appropriate place.
+
+Step 8: confirm the details given before clicking "Next"
+
+N/B the next page urges your to fill in your correct phone number as a code will be sent for confirmation.
+
+Step 9: fill in your correct Phone number.
+
+N/B A six-digit code will be sent to you for verification, fill it in before the time elapses.
+
+Step 10: fill in the six-digit code and click on the "next" button.
+
+Step 11:
+
+Input your house address.
+Appropriately fill in your state of residence, Local Government area, city, House number, and Street name on the allocated space for it. Click on next to move to another page.
+
+Congratulations, you have successfully created. Your account.
+
+Step 12: input your email address and password to sign in.
```

Comment:
This should be a take note kind of sentence, not another step, or you move the step upward before you say congratulations